types.go

```diff
@@ -165,6 +165,7 @@ type fwpmAction0 struct {
 }
 
 //go:notinheap
```

Comment:
Put this notinheap immediately adjacent to the type, just for style reasons.

You can expand the comment like:

```
// fwpmFilter0 is blah blah blah.
// See https://urlurlurl
//
//go:notinheap
type ...
```